compiler/pash_runtime.sh

```diff
@@ -140,10 +140,10 @@ else
         fi
         
         # Get assigned process id
-        response_args=($daemon_response)
+        response_args=("$daemon_response")
```

Comment:
This is a mistake. We want this to be a bash array and so we need to preserve the spaces. @a5ob7r could you revert this? This is the reason why the CI fails.